rps.js

```diff
@@ -0,0 +1,23 @@
+const game = function(userChoice) {
+
+    let compChoice = Math.random();
+    if (compChoice < 0.30){
+     compChoice = "rock";
+    } else if (compChoice <= 0.70){
+     compChoice= "paper";
+    }
+    else { 
+     compChoice = "scissors" ;
+    }
+    
```

Comment:
you can use `Math.floor()` with  `Math.random()` to get a random integers.

- first use `Math.random()` to get a random number from `0 to 0.99..` 
- if you multiply it with 3 `Math.random()*3` you will get random numbers from `0 to 2.99..` .
- and if you use `Math.floor()` it will give you integer that equal or less than the given number so `Math.floor(2.987) will return `2`.
- so if you write this `Math.floor( Math.random() * 3)` you will get a random number out of these numbers `0`, `1`, and `2`.